docs/user/integration/sap-cloud-logging/README.md

```diff
@@ -122,7 +133,7 @@ To enable log shipment to the SAP Cloud Logging service instance, follow this pr
     EOF      
     ```
 
-   By default, Kyma sets Istio access logs to disabled. To enable Istio access logs selectively for your workload, follow [Enable Istio access logs](https://kyma-project.io/#/istio/user/operation-guides/02-30-enable-istio-access-logs).
+1. By default, Kyma sets Istio access logs to disabled. To enable Istio access logs selectively for your workload, follow [Enable Istio access logs](https://kyma-project.io/#/istio/user/operation-guides/02-30-enable-istio-access-logs).
```

Comment:
Maybe we should change the order and first enable access logging in istio and then setup the pipeline to collect them. Then people will not overlook it and it feels like a more natural order